explorer/explorer_android.java

```diff
@@ -21,15 +21,19 @@
 import java.io.ByteArrayOutputStream;
 import java.io.Closeable;
 import java.io.Flushable;
+import java.util.ArrayList;
+import java.util.List;
 
 public class explorer_android {
     final Fragment frag = new explorer_android_fragment();
 
-    static int CODE_OPEN = 756456;
-    static int CODE_CREATE = 756457;
+    static List<Integer> import_codes = new ArrayList<Integer>();
+    static List<Integer> export_codes = new ArrayList<Integer>();
 
-    static public native void OpenCallback(InputStream f);
-    static public native void CreateCallback(OutputStream f);
+    static int PREFIX_CODE = 756456;
```

Comment:
Why are we adding this to the activity ID? I looked at [these docs](https://developer.android.com/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)), and I couldn't see anything indicating why this would be necessary. What am I missing? :D

Reply:
Makes sense to change to the `id` from `explorer` directly. The reason to use `PREFIX` was to prevent some "collision" with another task/activity that may call `onActivityResult`. So, if another library uses `id = 0` and explorer also uses `id = 0` I thought that it may cause problems. 

_I'm not sure if it's possible to have such "collision", but that is why have the "PREFIX"._

Reply:
I agree, let's use the explorer's ID directly. That is extremely unlikely to collide with anything, and it is simple to do.

Reply:
Fixed on `explorer: [android] remove id prefixes`.